src/main/pages/che-7/end-user-guide/proc_using-different-types-of-storage.adoc

```diff
@@ -60,3 +82,8 @@ Aslo two new properties added to `che.properties` that can be used to configure
 
 `che.workspace.storage.preferred_type`: this configuration property defines a default value for storage type that clients like Dashboard should propose for users during workspace creation/update. The 'async' value not recommended as a default type since it's still experimental (e.g. `che.workspace.storage.preferred_type=persistent`)
 
+The Storage Type switcher is available on the *Create Custom Workspace* page of the User Dashboard:
+
+image::troubleshooting/create-custom-ws-storage-type-2.png[link="{imagesdir}/troubleshooting/create-custom-ws-storage-type-2.png"]
```

Comment:
Is this image really belongs to troubleshooting area? Maybe https://github.com/eclipse/che-docs/tree/master/src/main/images/workspaces is a bit better place?

Reply:
as you wish